### PR TITLE
Completed Hero Section component

### PR DESCRIPTION
**Pull request by HassanShanJava**
Hi, I completed the hero section, I followed based on the Figma design.

Here is how the homepage looked before my work:
before

And this is how it will look like if you accept my work, also it is mobile responsive:
after-mobile

after

 
Note:
Do let me know if there need to be any changes made, I'm open to feedback.

### CODE REVIEW

File: src/components/heroSection/Hero.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+              <span className="text-[#A38DFD]">Engineers</span> come to find one
+              another !
+            </p>
+            <p className="py-6 text-base">
+              So many jobs available, all you have to do is keep up with our
+              posts. Check below for recent job openings.{" "}
+            </p>
+            <div className="bg-white p-2  flex justify-between items-center rounded-lg">
+              <div className="flex items-center">
+                <Image src={Search} alt="/" />
```
Comment (chadstewart) [inline src/components/heroSection/Hero.tsx:23]: Please change this from using an image to use the icon component in the atoms folder. This project is currently using React Icons for all it's Icons and so please use a Search Icon from that library. There are example usages of the Icon component in the footer to help you use it. Also delete the search svg once you've finished the immigration please.

Also, could you add the alt text, 'Search Icon' when you make that change please.

File: src/components/heroSection/Hero.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+              </div>
+              <div>
+                <button className="bg-[#165C9C] text-white font-bold p-3 rounded-xl">
+                  Search
+                </button>
+              </div>
+            </div>
+          </div>
+          <div className="max-h-[529px] max-w-[481px] pt-5 lg:pt-0">
+            <Image src={HeroImage} alt="/" />
```
Comment (chadstewart) [inline src/components/heroSection/Hero.tsx:38]:
```suggestion
            <Image src={HeroImage} alt="Hero Image" />
```

Could you please add the text 'Hero image' to this image tag please?

**Review by chadstewart — CHANGES_REQUESTED** [review]
First of all, great job on this issue! It's coming along quite nicely! I've only requested a few changes for this PR.

First of all, this project uses the Atomic Design architecture for our Front-End components. The hero component is currently not in any of those folders. I believe in this case it would be a molecule so please move the hero section component into the src/components/molecules folder. It helps us keep track of the different types of components we have and their complexity.

Secondly, we have a Storybook instance for this project. We already have several stories for this project. They should be under src/stories. In the molecules folder in the stories folder, could you add a story for this component? There are several examples in that folder to help you create a story!

That's about it! Thanks again for the contribution and looking forward to seeing the changes!

**Comment by HassanShanJava** [comment]
Sure, I get right on it!.

**Review by HassanShanJava — COMMENTED** [review]
Hi, I made all your suggested changes.
- Alt message on image tags
- File Structuring
- Added Stories for the hero section
- Used react icons similar to how its used in footer section

Please have a look at it and let me know if there's any issue.

File: src/pages/index.tsx [hunk 3]
```diff
@@ -1,8 +1,11 @@
+
 import WithPageLayout from "interfaces/with-page-layouts";
 import DefaultLayout from "components/templates/layouts/default-layout";
 import TwitterFeed from "components/organisms/twitter-feed/twitter-feed";
 import Head from "next/head";
 
+import { HeroStory } from "stories/molecules/hero.stories";
```
Comment (chadstewart) [inline src/pages/index.tsx:7]: Change this back to importing from the components folder. The stories are meant to document the component but aren't meant to be used in the project itself. The way you had it previously was good.

File: src/pages/index.tsx [hunk 4]
```diff
@@ -17,6 +20,8 @@ const Home: WithPageLayout<HomeProps> = () => {
           content="TechIsHiring - A space where great jobs and great devs can find each other!"
         />
       </Head>
+      {/* <Hero /> */}
+      <HeroStory />
```
Comment (chadstewart) [inline src/pages/index.tsx:24]:
```suggestion
      <Hero />
```

Change this back to the way it was.

**Review by chadstewart — CHANGES_REQUESTED** [review]
Almost there! Just a few minor things that needs to be changed!

File: src/components/heroSection/Hero.tsx (hunk 1 above)
Comment (chadstewart) [reply src/components/heroSection/Hero.tsx]: Just a quick reminder to change the this search image to an icon please? Thank you so much!

**Comment by HassanShanJava** [comment]
Hi, I made your suggested changes. 

Do let me know if there is anything else that needs to change!

**Review by chadstewart — APPROVED** [review]
> Hi, I made your suggested changes.
> 
> Do let me know if there is anything else that needs to change!

These are great! Thanks so much for all the work! I'll merge your contribution in a bit!

**Pull request merged**